app/views/tag/_advanced_tagging.html.erb

```diff
@@ -10,15 +10,15 @@
   <% end %>
   <div class="dropdown-divider"></div>
   <% if @node && @node.type == "page" %>
-    <a class="dropdown-item" href="#" data-value="parent:">Link to a "parent" page (wikis only)</a>
-    <a class="dropdown-item" href="#" data-value="style:">Add style</a>
-    <a class="dropdown-item" href="#" data-value="lang:">Add Translation widget</a>
+    <a class="dropdown-item" href="javascript:window.location.reload(true)" data-value="parent:">Link to a "parent" page (wikis only)</a>
```

Comment:
I think this may not work in Firefox, which is a little stricter on href javascript. You could put it in an `onclick` attribute, maybe?